.github/workflows/wda-package.yml

```diff
@@ -0,0 +1,48 @@
+name: Building WebDriverAgent
+
+on:
+  workflow_dispatch:
+  pull_request:  # for testing
+
+jobs:
+  build:
+    name: Build and analyse default scheme using xcodebuild command
+    runs-on: macos-latest
+
+    env:
+      XCODE_VERSION: 13.2.1
```

Comment:
is this supposed to be a minimum supported xcode version?

Reply:
Either is fine as this creating package's purpose. I mean apps generated newer Xcode can work with lower Xcode hosted macOSs. (I haven't seen an issue like Xcode 13 built one on Xcode 11 hosted macOS) Our CI already has test code to check minimum version, so this can be any version exists on a host. I have picked up current default version in `-latest` to make this explicit.

Only Xcode 12+ or Xcode 11- was an issue since they had different bundle id generating rule, which might affect package's info.plist. potentially it confuses users.